desafio-02/phewill/README.md

```diff
@@ -0,0 +1,15 @@
+### [William Garcia](https://github.com/phewill)
```

Comment:
Mais um arquivo que pode morrer. :) Voce só precisa de README.md dentro do seu diretorio (desafios-XX/phewill/...) *se* precisar passar instruções especiais pro leitor. Não é o caso aqui. Coloque as informações como autor e links pro repl.it dentro do próprio código (comentários no início do arquivo).